.github/workflows/static-analysis.yml

```diff
@@ -48,6 +48,41 @@ jobs:
     - name: Check for Detekt Issues with Type Resolution
       run: ./gradlew --stacktrace detektMain detektTest
 
+  eslint:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v4
+
+    - name: Install Node
+      uses: actions/setup-node@v4
+      with:
+        node-version: 20
+
+    - uses: pnpm/action-setup@v4
+      name: Install pnpm
+      with:
+        version: 9
+
+    - name: Get pnpm store directory
+      shell: bash
+      run: |
+        echo "STORE_PATH=$(pnpm store path --silent)" >> $GITHUB_ENV
+
+    - uses: actions/cache@v4
+      name: Setup pnpm cache
+      with:
+        path: ${{ env.STORE_PATH }}
+        key: ${{ runner.os }}-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
+        restore-keys: |
+          ${{ runner.os }}-pnpm-store-
+
+    - name: Install dependencies
+      run: pnpm -C ui install
+
+    - name: Run ESLint
+      run: cd ui && pnpm lint 
```

Comment:
Nit: This could / should use the `-C ui` from line 81.

Reply:
Ah, tried with `-c` and wondered why it didn't work, let's go with `-C`.